scripts/core/editor3/reducers/suggestions.jsx

```diff
@@ -245,6 +246,27 @@ const createChangeBlockStyleSuggestion = (state, {blockType, data}) => {
     return saveEditorStatus(state, editorState, 'change-block-type', true);
 };
 
+
+/**
+ * @ngdoc method
+ * @name isMergeParagraph
+ * @param {Object} editorState
+ * @param {Object} data - info about the author of suggestion
```

Comment:
How about naming the argument `authorInfo` instead of `data`, so there's no need to comment on what it contains

Reply:
It is suggestion related data, it is not only about author.

Reply:
How about suggestionAuthorInfo? :smile_cat:

Reply:
I think this is all around the code, it's just metadata for the suggestion, but I think currently it only hold the author info

Reply:
For block style contains also the set style. Probably suggestionData or suggestionInfo is a better name.
Anyway all over the app is used data so it is better this change to be performed in a different PR.